sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -508,4 +511,13 @@ object FunctionRegistry {
       new ExpressionInfo(clazz.getCanonicalName, name)
     }
   }
+
+  private def expressionGeneratorOuter[T <: Generator : ClassTag]
+    (name: String): (String, (ExpressionInfo, FunctionBuilder)) = {
```

Comment:
Nit: Could you follow the indentation style? https://github.com/databricks/scala-style-guide#spacing-and-indentation
```Scala
private def expressionGeneratorOuter[T <: Generator : ClassTag](
    name: String): (String, (ExpressionInfo, FunctionBuilder)) = {
  val (_, (info, generatorBuilder)) = expression[T](name)
  ...
```